src/style.css

```diff
@@ -1,3 +1,84 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+}
+
+ul li {
+  margin: 0;
+  padding-left: 20px;
+}
+
+ul :first-child {
+  padding: 0%;
```

Comment:
This selector might not work as expected for selecting the first child of `ul`. If you want to select the first `li` inside `ul`, the correct selector is `ul > li:first-child`.